test/test_baseline_trainers.py

```diff
@@ -401,7 +401,7 @@ def pause_at(self, idx):
 
     expected = sorted(set(range(num_envs)) - set(envs_to_pause))
 
-    assert envs._running == expected
+    assert envs._running == expected  # type:ignore
```

Comment:
Can this be a qualified type ignore? ie. `# type:ignore[ignore_reason]` instead of a blanket one?